src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController+Extension.swift

```diff
@@ -136,8 +136,8 @@ extension OnboardingInfoViewController {
 			topSeparator.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor),
 			topSeparator.heightAnchor.constraint(equalToConstant: 1),
 			containerStackView.topAnchor.constraint(equalTo: topSeparator.bottomAnchor, constant: 12),
-			containerStackView.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor, constant: 18),
-			containerStackView.trailingAnchor.constraint(equalTo: layoutMarginsGuide.trailingAnchor, constant: 18),
+			containerStackView.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor),
+			containerStackView.trailingAnchor.constraint(equalTo: layoutMarginsGuide.trailingAnchor),
```

Comment:
This might match the requirements @marcussc got informed from UX - I hope
```suggestion
			containerStackView.leadingAnchor.constraint(equalTo: layoutMarginsGuide.leadingAnchor, constant: 18),
			containerStackView.trailingAnchor.constraint(equalTo: layoutMarginsGuide.trailingAnchor, constant: -18),
```